Makefile

```diff
@@ -94,76 +94,65 @@ images = pgo-apiserver \
 	crunchy-postgres-exporter \
 	postgres-operator
 
-.PHONY: all installrbac setup setupnamespaces cleannamespaces bounce \
-	deployoperator runmain runapiserver cli-docs clean push pull \
-	release default
+.PHONY: all installrbac setup setupnamespaces cleannamespaces \
+	deployoperator cli-docs clean push pull release
 
 
 #======= Main functions =======
 all: linuxpgo $(images:%=%-image)
 
 installrbac:
-	cd deploy && ./install-rbac.sh
+	PGOROOT='$(PGOROOT)' ./deploy/install-rbac.sh
 
 setup:
-	./bin/get-deps.sh
+	PGOROOT='$(PGOROOT)' ./bin/get-deps.sh
 
 setupnamespaces:
-	cd deploy && ./setupnamespaces.sh
+	PGOROOT='$(PGOROOT)' ./deploy/setupnamespaces.sh
 
 cleannamespaces:
-	cd deploy && ./cleannamespaces.sh
-
-bounce:
-	$(PGO_CMD) \
-		--namespace=$(PGO_OPERATOR_NAMESPACE) \
-		get pod \
-		--selector=name=postgres-operator \
-		-o=jsonpath="{.items[0].metadata.name}" \
-	| xargs $(PGO_CMD) --namespace=$(PGO_OPERATOR_NAMESPACE) delete pod
```

Comment:
With this section gone, can `PGO_OPERATOR_NAMESPACE` be removed from the Makefile?

Reply:
Yep. Looks like `PGO_CMD` can go, too.